grpc/nodejs/BUILD

```diff
@@ -23,28 +23,13 @@ load("@vaticle_dependencies//distribution:deployment.bzl", "deployment")
 load("//:deployment.bzl", github_deployment = "deployment")
 load("//grpc/nodejs:rules.bzl", "ts_grpc_compile")
 
-
-nodejs_binary(
```

Comment:
we no longer need to wrap the nodejs binaries since we use the ones from `protoc-gen-ts`'s bazel rules directly